src/plugins/util/web.js

```diff
@@ -158,26 +161,33 @@ function addRequestTags (req) {
 
 function addResponseTags (req) {
   const span = req._datadog.span
-  const tags = span.context().tags
   const res = req._datadog.res
 
-  if (!tags.hasOwnProperty(HTTP_ROUTE) && req._datadog.paths.length > 0) {
+  if (req._datadog.paths.length > 0) {
```

Comment:
`HTTP_ROUTE` basically wouldn't ever get set here right? since this happens before hooks are called?

Reply:
*get set before here

Reply:
It's possible, but it's possible for all other tags as well. The idea is that you should not rely on the span from the active scope to set this kind of tag because it may change (i.e. if we add middleware and route spans)

Reply:
One thing I was wondering is if we should never override any tag set by the user, but that would be a major change since we set tags early in many cases.

Reply:
thanks :+1: